Makefile

```diff
@@ -42,7 +42,8 @@
 #   - docker-tag-stable - re-tags the images made by 'make docker' with the :stable tag
 #   - help-docs - generate the command reference docs
 
-export SAMPLECONFIG_DIR :=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))/sampleconfig
+export PROJECT_ROOT_DIR :=$(dir $(realpath $(firstword $(MAKEFILE_LIST))))
+export SAMPLECONFIG_DIR :=$(PROJECT_ROOT_DIR)/sampleconfig
```

Comment:
Unnecessary and puts a dependency on the make process instead of the go toolchain.

We have gotten to the point where we can use standard go tools without project path specific environment variables to the the majority of our work. There are a few niggles (like installing tools) but they don't impact every test execution.